proposals/build-cli-options.md

```diff
@@ -0,0 +1,50 @@
+# [Proposal] Allow additional cli options to pass to the build command
+
+Related to:
+- https://github.com/devcontainers/spec/discussions/301
+- https://github.com/devcontainers/spec/pull/324
+- https://github.com/microsoft/vscode-remote-release/issues/3545
+
+## Goal
+
+Extend the build section of the configuration with an option to pass additional arguments via cli.
+
+## Proposal
+
+Introduce the property `cliOptions` to the `buildOptions` in the schema as type `Array (string)`. This field contains elements that are directly passed as cli-arguments to the `build` command. It is similar to `runArgs` property which contains cli-arguments passed to the `run` command.
```

Comment:
Should this be "`cliArgs` to the `build` property" to match existing names?

Reply:
Adjusted it, yes it's the `build` property (the type in the schema is `buildOptions`).